### ISSUE TITLE
Imports for reduced immunization module includes columns that are part of the extensive module

### ISSUE THREAD

**Issue by kwa20** [hzi-braunschweig/SORMAS-Project#8119]
### Bug Description

The imoprt templates include columns for vaccinations that are excluded in the reduced immunization module. Pregnancy information can also be imported that way while health condition information can be entered but won't get imported successfully.
### Steps to Reproduce

1. Enable the reduced immunization module
2. Go to either case directory, contact directory or event participant list and download the import templates
3. check the templates for vaccination information
4. Enter all available vaccination related fields and the respective compulsory entitiy fields
5. Import the file
6. Check the import for the reduced vaccination module
7. Enable the extensive immunization module and check the imported vaccinations

### Expected Behavior

When the reduced immunization modue is enabled, health condition and pregnancy related columns should not be availbale in the import templates.

### Screenshots

![vaccImport2](https://user-images.githubusercontent.com/70317594/155497506-e98b6354-1405-45ae-9895-303ca158a806.PNG)
![vaccImport](https://user-images.githubusercontent.com/70317594/155497504-d40509e2-22ff-4cb3-8b68-6a663c30eeea.PNG)

### System Details

* Device:
* SORMAS version: 1.68.0
* Android version/Browser: chrome
* Server URL: test1401.sormas-oegd.de
* User Role: admin, national user

**Comment by MateStrysewske** [comment]
@kwa20 Not a bug in my eyes because this was never specified.

**Comment by kwa20** [comment]
@MateStrysewske https://github.com/hzi-braunschweig/SORMAS-Project/issues/7532 only mentions the variables that are actually available in the reduced immunization module. This might also have data security implications

**Comment by JaquM** [comment]
@MateStrysewske exactly, it was not specified to have these fields for the reduced vaccination module, so having them is a bug.

**Comment by abrudanancuta** [comment]
Validated on web local environment with the version 1.70.0-SNAPSHOT (https://github.com/hzi-braunschweig/SORMAS-Project/commit/182e25533d54ace1733c23b1eab267edbbe407a9)
